### PR TITLE
docs - gpcopy add  --parallelize-leaf-partitions option

### PR DESCRIPTION
**Pull request by mkiyama**
Will be backported to 5X_STABLE

### CODE REVIEW

**Comment by mkiyama** [comment]
@BrianPivotal 
Is there a PR or tracker story for this feature?
How does a user perform this task?

**Comment by dyozie** [comment]
@mkiyama - if you have a test system available, can you try just specifying a leaf partition name with --include-table or --include-table-file?

**Comment by mkiyama** [comment]
@dyozie gpcopy is an external component.
I haven't been able to build it, and it doesn't have a daily build like GPDB.

**Comment by dyozie** [comment]
Yeah - I guess from Brian's statement it does seem like this was recently added to the code.

File: gpdb-doc/dita/utility_guide/admin_utilities/gpcopy.xml [hunk 1]
```diff
@@ -340,6 +341,15 @@
                     <pd>For smaller tables, copying tables using the Greenplum Database master is
                         more efficient than using segment instances.</pd>
                 </plentry>
+                <plentry>
+                    <pt>--parallelize-leaf-partitions</pt>
+                    <pd>If specified, the utility copies the leaf partition tables of a partitioned
+                        table in parallel. The default is to copy the partitioned table as a single
+                        table based on root partition table. </pd>
```
Comment (dyozie) [inline gpdb-doc/dita/utility_guide/admin_utilities/gpcopy.xml]: on root -> on the root

File: gpdb-doc/dita/utility_guide/admin_utilities/gpcopy.xml [hunk 2]
```diff
@@ -495,8 +505,12 @@
                 <p><b>Copying Partitioned Tables</b></p>
                 <p>When copying data for a partitioned table, if a leaf partition has been exchanged
                     with an external table, that leaf partition is created, but data is not copied. </p>
-                <p>You cannot copy an individual leaf partition, you must copy the entire
-                    partitioned table.</p>
+                <p>If you specify copying leaf partitions of a partitioned table with an option such
+                    as <codeph>--include-table</codeph> or <codeph>--exclude-table</codeph>,
+                        <codeph>gpcopy</codeph> creates the partitioned table if it does not exist.
+                    Only the data for the specified leaf partitioned are added to the partitioned
```
Comment (dyozie) [inline gpdb-doc/dita/utility_guide/admin_utilities/gpcopy.xml]: partitioned -> partitions

**Pull request merged**